connector/connectionProperties.js

```diff
@@ -0,0 +1,14 @@
+(function propertiesbuilder(attr) {
+    var props = {};
```

Comment:
The ADL docs indicate the `compressors` and `authSource` properties should also be included ([see here](https://docs.mongodb.com/datalake/tutorial/jdbc-driver/#configure-the-driver-for-your-tableau)). I've yet to test _without_ those, so maybe we should check before including them. However, if they are necessary then we'll need to include them here.

The JDBC driver takes those properties (and any others in the props file) and adds them to the connection string.

Reply:
Included `compressors` and `authSource` and confirmed Tableau connects and can query db metadata and data.

Reply:
Maybe this project would be a good time to audit how necessary these two properties are for ADL. We could ask Tom or Ben if they have any idea on this.

Reply:
Removed `compressors` as it is ignored by ADL.  `authSource` is used, keeping that property.